src/dev/flang/fe/Module.java

```diff
@@ -316,8 +318,9 @@ protected boolean featureVisible(SourceFile usedIn, AbstractFeature af)
     var definedIn = af.pos()._sourceFile;
     var v = af.visibility();
 
-    return  // built-in or generated features like #loop0
-            af.pos().isBuiltIn()
+    return  SourcePosition.builtIn._sourceFile == usedIn
```

Comment:
This needs an explanation comment like the other cases. 
It would be better to not compare against `SourcePosition.builtIn._sourceFile`, but instead add a new static field `SourcePosition._builtInFile_`.
Then, I would still restrict this a bit more, e.g.,
```
 // generated inline code may access base library features
|| v.ordinal() >= Visi.MOD.ordinal() && m.name.equals("base") && usedIn == SourcePosition._builtInFile_
```
(and "base" should better be a `static final` field in `FrontEnd.java`).